helmfile/charts/harbor/harbor-backup/scripts/harbor-backup.sh

```diff
@@ -25,26 +28,23 @@ wait_for_db_ready() {
 
 dump_database() {
     echo "Dumping database"  >&2
-    pg_dump -U postgres -h "$PG_HOSTNAME" registry > ${backup_dir}/registry.back
-    pg_dump -U postgres -h "$PG_HOSTNAME" postgres > ${backup_dir}/postgres.back
-    pg_dump -U postgres -h "$PG_HOSTNAME" notarysigner > ${backup_dir}/notarysigner.back
-    pg_dump -U postgres -h "$PG_HOSTNAME" notaryserver > ${backup_dir}/notaryserver.back
+    pg_dump -U postgres -h "$PG_HOSTNAME" registry | gzip -c > "${dump_dir}/registry.back.gz"
+    pg_dump -U postgres -h "$PG_HOSTNAME" postgres | gzip -c > "${dump_dir}/postgres.back.gz"
```

Comment:
We need to add this commit as well https://github.com/elastisys/compliantkubernetes-apps/commit/3abe5f393ebfca77bc3a4a1ae85581fd8b5c5a2f to make it work for apps < 0.35

Reply:
Done !!